content/docs/faq-styling.md

```diff
@@ -0,0 +1,35 @@
+---
+id: faq-styling
+title: Styling and CSS
+permalink: docs/faq-styling.html
+layout: docs
+category: FAQ
+---
+
+### How do I add CSS classes to components?
+
+Pass a string as the `className` prop:
+
+```jsx
+render() {
+  return <span className="menu navigation-menu">Menu</span>
+}
+```
+
+### Can I use inline styles?
+
+Yes, see https://reactjs.org/docs/dom-elements.html#style
```

Comment:
For links within the website, I think it's nicer to use a shorter format rather than the full URL. For example:

> Yes! [Read here](/docs/dom-elements.html#style) to learn more.